### ISSUE TITLE
Are building type presets really necessary in iD?

### ISSUE THREAD

**Issue by AntMadeira** [openstreetmap/iD#7774]
Hi there.
iD has two presets that can be a bit confusing, specially for newbies: `amenity=place_of_worship` + `religion=christian` and `building=church`. (it could be extended to other types of buildings)
In a country with many thousands of churches, this is problematic, because most casual mappers don't read OSM wikis.
To avoid having dozens or hundreds of actual churches mapped only with building=church, why not remove those presets and use only the type dropdown selector that already exists when adding `amenity=place_of_worship` + `religion=christian`?

**Comment by quincylvania** [comment]
@AntMadeira I agree some of the building presets can be confusing for many mappers. I'd consider disabling them by default if these accommodations were in place:

- The building type field was translatable (#2708)
- A building "preset pack" could be enabled for mappers who care a lot about these

**Comment by AntMadeira** [comment]
If the type of building was translatable (that is another issue that I would love to be implemented, since it would help pick the correct keys) I agree it should be removed from presets. People could choose the building type from the field inside other main presets. As it is now, it causes a lot of unnecessary doubts.

I didn't understand what you meant with "preset pack"...

**Comment by quincylvania** [comment]
> I didn't understand what you meant with "preset pack"...

People currently see all available presets at once, mixed together. We could separate some groups of specialized presets out into "add-on packs" that advanced mappers could enable in a separate menu or via a URL parameter.

**Comment by AntMadeira** [comment]
Thank you for your clarification.
Don't know which presets would be included in those packs, but it seems a good idea to separate basic/common presets from more advanced/complex ones.
Still, I believe that the building type specifically should be only selectable only when you choose the building preset, or some kind of amenity like school, hospital, place_of_worship, etc.
